public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -157,7 +160,7 @@ export const DashNav = React.memo<Props>((props) => {
     }
 
     if (dashboard.meta.publicDashboardEnabled) {
-      buttons.push(<Tag name="Public" colorIndex={5}></Tag>);
+      buttons.push(<Tag name="Public" colorIndex={5} aria-label={selectors.publicDashboardTag}></Tag>);
```

Comment:
Please use `data-testid` for selectors for tests. https://github.com/grafana/grafana/pull/36483

only use aria-label for actual labels that you would make visible in the UI.